lib/msal-browser/src/cache/TokenCache.ts

```diff
@@ -142,7 +143,8 @@ export class TokenCache implements ITokenCache {
                 this.config.system.networkClient,
                 this.storage,
                 authorityOptions,
-                this.logger
+                this.logger,
+                request.correlationId || BrowserCrypto.createNewGuid()
```

Comment:
Is request.correlationId an option to the API? This is used for sideloading tokens so I assume this is not always set?

Reply:
Correct on both counts